README.md

```diff
@@ -294,7 +294,7 @@ same major line. Should you need to upgrade to a new major, use an explicit
   empty password, explicitly set `COREPACK_NPM_PASSWORD` to an empty string.
 
 - `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` are supported through
-  [`node-proxy-agent`](https://github.com/TooTallNate/node-proxy-agent).
+  [`proxy-from-env`](https://github.com/Rob--W/proxy-from-env).
```

Comment:
I think we should be a bit more specific here.
We use `proxy-from-env` to get which proxy address to pass to Undici.

Reply:
Does it matter?

Reply:
Not really, no.
Feel free to ignore this.